components/03-modules/popup/_popup.scss

```diff
@@ -28,14 +33,30 @@ $popup__arrow-transform : rotate(45deg) !default;
         width: 100vw;
     }
 
-    &__arrow {
+    &__handler {
+        position: relative;
+        margin-top: $popup__content-top-margin;
+    }
+
+    &__close-btn {
```

Comment:
Are we using shorted `btn` instead of `button` ?

Reply:
IMO no difference if you knew what the element the class was described ;), changed for `button`

Reply:
ok, thanks